lambdas/retrieve-cri-oauth-access-token/src/main/java/uk/gov/di/ipv/core/retrievecrioauthaccesstoken/RetrieveCriOauthAccessTokenHandler.java

```diff
@@ -140,24 +135,8 @@ public APIGatewayProxyResponseEvent handleRequest(
             BearerAccessToken accessToken =
                     credentialIssuerService.exchangeCodeForToken(
                             request, credentialIssuerConfig, apiKey);
-            List<SignedJWT> verifiableCredentials =
-                    credentialIssuerService.getVerifiableCredential(
-                            accessToken, credentialIssuerConfig, apiKey);
-
-            for (SignedJWT vc : verifiableCredentials) {
-                verifiableCredentialJwtValidator.validate(vc, credentialIssuerConfig, userId);
-
-                sendIpvVcReceivedAuditEvent(auditEventUser, vc);
-
-                if (configurationService.isNotRunningInProd()) {
-                    LOGGER.info("Submitting VC to CI storage system");
-                    submitVCAndSwallowErrors(vc, clientSessionDetailsDto.getGovukSigninJourneyId());
-                }
-
-                credentialIssuerService.persistUserCredentials(
-                        vc.serialize(), request.getCredentialIssuerId(), userId);
-            }
 
+            updateIpvSessionItemAccessToken(ipvSessionItem, accessToken);
```

Comment:
Could we refactor these 2 methods here slightly. I think it could be a bit confusing at first glance to see when the actual ipvSessionItem is being updated. As this method just adds the access token to the item, then the next is where we actually update it in Dynamo.

I can imagine the scenario that further down the line we swap the order or these methods but we could miss that fact that it would stop the access token being updated into Dynamo. If that makes sense?

Maybe having the 2 methods just update the ipvSessionItem. Then a separate statement below here to do the actual `ipvSessionService.updateIpvSession(ipvSessionItem);` call? Also maybe updating the method names to setIpvSessionItemAccessToken and setIpvSessionVisitedCredentials might be clearer to say they aren't doing an update request.

```suggestion
            setIpvSessionItemAccessToken(ipvSessionItem, accessToken);
            setIpvSessionItemVisitedCredentials(ipvSessionItem, request.getCredentialIssuerId(), true, null);
            
            ipvSessionService.updateIpvSession(ipvSessionItem);
```

Reply:
Valid points - I've updated the method names, and shifted `ipvSessionService.updateIpvSession(ipvSessionItem);` out of the second method, to below the two `set` methods.